youtube_dl/extractor/bigflix.py

```diff
@@ -0,0 +1,56 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from base64 import b64decode
+from urllib import unquote
+
+from .common import InfoExtractor
+from ..utils import sanitized_Request
+
+
+class BigflixIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?bigflix\.com/.*/(?P<id>[0-9]+)'
+    _TESTS = [
+        {
+            'url': 'http://www.bigflix.com/Hindi-movies/Action-movies/Singham-Returns/16537',
+            'md5': 'ec76aa9b1129e2e5b301a474e54fab74',
+            'info_dict': {
+                'id': '16537',
+                'ext': 'mp4',
+                'title': 'Singham Returns',
+                'description': 'md5:3d2ba5815f14911d5cc6a501ae0cf65d',
+            }
+        },
+        {
+            'url': 'http://www.bigflix.com/Tamil-movies/Drama-movies/Madarasapatinam/16070',
+            'md5': '5a62a47e79523bebe1a727a907fdf2b8',
+            'info_dict': {
+                'id': '16070',
+                'ext': 'mp4',
+                'title': 'Madarasapatinam',
+                'description': 'md5:63b9b8ed79189c6f0418c26d9a3452ca',
+            }
+        },
+    ]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+
+        request = sanitized_Request(url)
```

Comment:
No need to use `sanitized_Request` request here, pass url directly to download method.